### PR TITLE
Initial Kube rollout of registry

### PR DESCRIPTION
**Pull request by dasmb**

### CODE REVIEW

File: deploy/registry/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+metadata:
+  name: {{ $fullName }}-{{ $stage }}
+  labels:
+    helm.sh/chart: {{ include "deploy.chart" $ }}
+    app.kubernetes.io/instance: {{ $.Release.Name }}
+    app.kubernetes.io/managed-by: {{ $.Release.Service }}
+    app.kubernetes.io/name: {{ include "deploy.name" $ }}
+    app.kubernetes.io/stage: {{ $stage }}
+  annotations:
+    sumologic.com/sourceCategory: {{ $fullName }}_docker_logs
```
Comment (grahamar) [inline deploy/registry/templates/deployment.yaml]: I think you want this at the pod level right? Around here - https://github.com/flowcommerce/registry/pull/319/files#diff-51e48737a4ad28b41345a27c0dcea727R29

Comment (dasmb) [reply deploy/registry/templates/deployment.yaml]: Ack

File: deploy/registry/values.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+nameOverride: ""
+fullnameOverride: ""
+
+service:
+  port: 9000
+
+iamRole: arn:aws:iam::479720515435:role/ecsInstanceRole
+image:
+  repository: 479720515435.dkr.ecr.us-east-1.amazonaws.com/marketing-gateway
```
Comment (grahamar) [inline deploy/registry/values.yaml]: this repository will be `flowcommerce/registry` right?

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  }
+
+  agent {
+    kubernetes {
+      label 'worker-registry'
+      inheritFrom 'default'
+
+      containerTemplates([
+        containerTemplate(name: 'helm', image: "lachlanevenson/k8s-helm:v2.12.0", command: 'cat', ttyEnabled: true),
+        containerTemplate(name: 'scala', image: "jenkinsxio/builder-scala", command: 'cat', ttyEnabled: true),
```
Comment (grahamar) [inline Jenkinsfile]: You can drop the `scala` container since we're not doing CI in Jenkins any more

File: Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+      steps {
+        container('docker') {
+          docker.withRegistry('https://hub.docker.com', 'docker-hub-credentials') {
+            script {
+              IMAGE_TAG = "${env.BRANCH_NAME.toLowerCase()}-${env.BUILD_NUMBER}"
+              withCaching(cacheDirectories: ['/root/.sbt/boot', '/root/.ivy2']) {
+                image = docker.build("$ORG/$APP_NAME:$IMAGE_TAG", '-f Dockerfile .')
+              }
+            }
+            script { image.push() }
```
Comment (grahamar) [inline Jenkinsfile]: You can move the `image.push()` up into the script tag above to consolidate.

**Pull request merged**